### PR TITLE
Get MIEngine working after the refactor

### PR DESCRIPTION
**Pull request by gregg-miskelly**
- Update the name of the managed resource to match what is embedded in the module
- Update the MIDebugPackage and MICoreUnitTests projects to copy required facade assemblies
- Update the MIDebugPackage to exclude extra assemblies it was copying which we don't plan to ship with the MIEngine

### CODE REVIEW

**Comment by gregg-miskelly** [comment]
@chlowell please take a look. @chuckries you might want to look also, as we will need to follow this pattern as we add more dependencies moving forward.

File: src/MICoreUnitTests/MICoreUnitTests.csproj [hunk 1]
```diff
@@ -97,6 +98,9 @@
     <Compile Include="MIResultsTests.cs" />
     <Compile Include="NdkVersionTests.cs" />
     <Compile Include="Properties\AssemblyInfo.cs" />
+    <None Include="@(RequiredNetFX46FacadeAssemblies)">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
```
Comment (gregg-miskelly) [inline src/MICoreUnitTests/MICoreUnitTests.csproj]: This should probably be 'Always' instead

**Comment by chuckries** [comment]
:+1:

**Comment by chlowell** [comment]
Looks good to me :+1:

**Pull request merged**